index.html

```diff
@@ -2303,21 +2303,33 @@ <h3>Interface Definition</h3>
 };</pre>
         <section>
           <h2>Constructors</h2>
-          <p>When the constructor is invoked, the following steps MUST be run:</p>
+          When the <code><a>RTCDtlsTransport</a></code> constructor is invoked,
+          the user agent MUST run the following steps:
           <ol>
             <li><p>Let <var>transport</var> be the first argument.</p></li>
-            <li><p>If <code><var>transport</var>.state</code> is <code>closed</code>
-            <a>throw</a> an <code>InvalidStateError</code> and abort these steps.</p></li>
-            <li>Let <var>certificates</var> be the second argument.</li>
-            <li><p>If <var>certificates</var> is non-null, check that the
-            <code>expires</code> attribute of each <code><a>RTCCertificate</a></code>
-            object is in the future.  If a certificate has expired, <a>throw</a> an
-            <code>InvalidParameters</code> and abort these steps.</p></li>
-            <li><p>Let <var>dtlsTransport</var> be a new <code><a>RTCDtlsTransport</a></code>
-            object with <var>certificates</var>.</p></li>
-            <li><p>Let <var>dltsTransport</var> have
-            <dfn>[[\SendHeaderExtensions]]</dfn> and <dfn>[[\ReceiveHeaderExtensions]]</dfn>
-            internal slots initialized to <code>null</code>.</p></li>
+            <li><p>If <var>transport</var> is in the <code>closed</code>
+            state, <a>throw</a> an <code>InvalidStateError</code> and abort
+            these steps.</p></li>
+            <li><p>Let <var>certificates</var> be the second argument if provided, <code>null</code>
+            otherwise.</p></li>
+            <li><p>If the <var>certificates</var> is not <code>null</code> and is non-empty, check that the
+            <code>expires</code> attribute of each <code><a>RTCCertificate</a></code> object
+            is in the future. If a certificate has expired, <a>throw</a> an <code>InvalidAccessError</code>
+            and abort these steps.</p></li>
+            <li><p>Let <var>dtlstransport</var> be a newly constructed
+            <code><a>RTCDtlsTransport</a></code> object.</p></li>
+            <li><p>Let <var>dtlstransport</var> have a <dfn>[[\DtlsTransportState]]</dfn> internal slot,
+            initialized to <code>new</code>.</p></li>
+            <li><p>Let <var>dltsTransport</var> have <dfn>[[\SendHeaderExtensions]]</dfn>
+            and <dfn>[[\ReceiveHeaderExtensions]]</dfn> internal slots initialized to
+            <code>null</code>.</p></li>
+            <li><p>Let <var>dtlstransport</var> have a <dfn>[[\Certificates]]</dfn> internal slot.</p></li>
+            <li><p>If <var>certificates</var> is non-null and is non-empty,
+            initialize the <a>[[\Certificates]]</a> internal slot to <var>certificates</var>.</p></li>
+            <li><p>If <var>certificates</var> is <code>null</code> or is empty,
+            generate a certificate using the default ciphersuite and store this in the
```

Comment:
keygenalgorithm. I assume this is already fixed like in https://github.com/w3c/webrtc-quic/commit/1ace017d49e5d4d173a7aa26e17520d4a5495a00 but not pushed :-)